content/en/network_monitoring/devices/profiles.md

```diff
@@ -64,6 +64,29 @@ Profiles can be used interchangeably, meaning devices that share MIB dependencie
 
 For more Datadog provided profiles, see the [GitHub repository][3].
 
+### Metadata definition by profile
+
+Profiles have a metadata section that can be used to define where and how metadata should be collected. Values can be static or come from an OID value.
+Supported fields can be found [here][4].
```

Comment:
Is there a specific section (to replace `this section`) in the code that we can point users to?

```suggestion
See [this section][4] for the supported fields.
```

Reply:
would `See the [DeviceMetadata][4] for the supported fields` work ?
and I'll update the link to point to the section instead of the whole file

Reply:
Thanks! I updated the suggestion.

Reply:
addressed in [0bc14c3](https://github.com/DataDog/documentation/pull/21347/commits/0bc14c33ec658b543030acd309dd209296be0569)